config.xml

```diff
@@ -13,7 +9,9 @@
   </author>
   <content src="index.html"/>
   <access origin="*"/>
-  <allow-navigation href="https://em-game.eecs.berkeley.edu/*" />
+  <allow-navigation href="https://em-game.eecs.berkeley.edu/*"/>
+  <allow-navigation href="http://10.142.36.203:8100"/>
```

Comment:
Do you really need this `allow-navigation`? I think we only used it to access the em-game website directly from the app.